operations/experimental/use-logcache-syslog-ingress-windows2019.yml

```diff
@@ -1,4 +1,4 @@
+# Has been integrated into cf-deployment.yml
+#
+# Please delete this file in the future
```

Comment:
Since this is already a breaking change, you can delete this ops-file (and the corresponding entries in the `README.md` and `units/tests/experimental_test/operations.yml` files). We'll just note the deletion in the release notes.

Reply:
👍